cmd/goal/application.go

```diff
@@ -995,3 +1047,140 @@ var infoAppCmd = &cobra.Command{
 		}
 	},
 }
+
+var methodAppCmd = &cobra.Command{
+	Use:   "method",
+	Short: "Invoke a method",
+	Long:  `Invoke a method in an App (stateful contract) with an application call transaction`,
```

Comment:
I'm not proposing any changes here, but view this as a learning opportunity for me about Algorand's conventions. Cleary the convention in the file is for this field to use raw string literals. I guess that makes sense, because in theory the `Long` description could span several lines. However, none of the descriptions in this file seem to utilize any of the string literal's special features. Any thoughts? You can also totally ignore.

Reply:
I don't have specific thoughts or idea about it, just following the previous conventions. I just noticed the nearest editing for adding method/updating description is 16 months ago... and I suppose they might want to do something with string literal.